docs/maintain/maintain-guides-validator-payout.md

```diff
@@ -185,10 +190,11 @@ validator, then you will lose 1 DOT in each case.
 
 :::caution
 
-If a validator is oversubscribed in an era, staking rewards are distributed only to the the top 
-{{ polkadot_max_nominators }} nominators and the rest of the nominators do not receive any rewards. 
-This is not the case for slashing! Every active nominator of the validator committing slashable offence
-will be slashed.
+If a validator is oversubscribed in an era, staking rewards are distributed only to the the top
+{{ polkadot: <RPC network="polkadot" path="query.staking.maxNominatorsCount" defaultValue={50000}/> :polkadot }}
```

Comment:
The whole NPoS system on Polkadot can support only 22500 nominators :) 
This statement says it is will rewards the 50000 nominators of a validator! Need to use the constant below.

<img width="1430" alt="image" src="https://user-images.githubusercontent.com/86818441/188883197-d4b0a12e-1264-4f5d-b539-28f417cd1980.png">